resources/tests/setup_rootfs.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Script that customizes a rootfs for CI images.
+#
+
+create_basic_rootfs() {
+    BUILD_DIR="$1"
+    DISKMNT="$1/mnt/rootfs"
+    IMAGE="$3"
+    SSH_DIR="$BUILD_DIR/ssh"
+    RESOURCE_DIR="$2"
+    FLAVOUR="$4"
+
+    mount "$IMAGE" "$DISKMNT"
+
+    # Set a hostname.
+    echo "ubuntu-fc-uvm" > "$DISKMNT/etc/hostname"
+
+    # Setup fcnet service. This is a custom Firecracker
+    # setup for assigning IPs to the network interfaces in the guests spawned
+    # by the CI.
+    cp "$RESOURCE_DIR/fcnet-setup.sh"  "$DISKMNT/usr/local/bin/"
+    chmod +x "$DISKMNT"/usr/local/bin/fcnet-setup.sh
+    touch "$DISKMNT"/etc/systemd/system/fcnet.service
+cat >> "$DISKMNT"/etc/systemd/system/fcnet.service << EOF
```

Comment:
We should output to the file instead of appending (`cat > file.name`). If for some reason this file exists, the content will be appended.